packages/ffe-form-react/src/InputGroup.js

```diff
@@ -108,6 +111,8 @@ class InputGroup extends Component {
 const instanceOfComponent = component => shape({ type: oneOf([component]) });
 
 InputGroup.propTypes = {
+    /** The id that will be used on the input child if you don't want a generated one */
+    childId: string,
```

Comment:
Maybe we can call it `inputId` since we're going to use it on the input-control? An `<InputGroup />` has several children, including labels, tooltips etc.

Reply:
I agree, that's a much better name, I'll fix